.github/workflows/build-posix.yml

```diff
@@ -0,0 +1,26 @@
+name: Build (POSIX)
+
+on: [push, pull_request]
+
+jobs:
+  build-posix:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update APT package database
+        shell: bash
```

Comment:
FYI, you can do something like the following instead of specifying `shell: bash` for every step:
https://github.com/zephyrproject-rtos/sdk-ng/blob/79789b3d4650ccd3df318303573b9cd727745cd1/.github/workflows/ci.yml#L363-L365

also `shell: bash` should not be required (it is implied) given that this workflow runs on `ubuntu-latest`.

Reply:
For some reason, I can't use "source" in the shell area unless I explicitly set the shell to bash.